config/development.toml

```diff
@@ -69,12 +69,11 @@ common_merchant_identifier = "COMMON MERCHANT IDENTIFIER"
 applepay_endpoint = "DOMAIN SPECIFIC ENDPOINT"
 
 [locker]
-host = ""
-host_rs = ""
-mock_locker = true
-basilisk_host = ""
-locker_enabled = true
-ttl_for_storage_in_secs = 220752000
+host = "http://127.0.0.1:3000"
+host_rs = "http://127.0.0.1:3000"
+mock_locker = false
```

Comment:
Specifying `mock_locker = false` would force everyone running the application locally to set up the card vault. Is that the intended behavior?